modules/registry-authentication-enabled-registry-overview.adoc

```diff
@@ -36,9 +36,7 @@ to stand-alone projects outside {product-title}.
 You can use `podman login` with your credentials, either username and password
 or authentication token, to access content on the new registry.
 
-All imagestreams point to the new registry. Because the registry requires
-authentication for access, the Samples Operator creates the
-`samples-registry-credentials` secret.
+All imagestreams point to the new registry, which uses the installation pull secret to authenticate.
```

Comment:
This change looks like exclusive with the following part, it need update too.
`You must place your credentials in two places:`
   `OpenShift namespace. Your credentials must exist in the OpenShift namespace so that the imagestreams in the OpenShift namespace can import.`
@gabemontero

Reply:
My understanding from @dmage is that you can put the credential in one of two places when you want to import sample imagestreams from a custom external registry that requires credentials

either 1) as a dockerconfigjson secret in the openshift namespace
or 2) on the host / node

so as long as @xiuwang meant "either" and not "both", and @dmage does not correct us here, I'm fine with adding this additional mount of detail @bmcelvee

Reply:
`You must place your credentials in two places:` isn't the best way to do it since 4.5.
It's better to change `openshift-config/pull-secret` and wait until operators distribute it throughout the cluster.

Reply:
I went ahead and changed this to "either". If it needs to be changed back, I can do so in a follow-up PR.